packages/twenty-server/src/engine/twenty-orm/factories/workspace-datasource.factory.ts

```diff
@@ -78,18 +78,43 @@ export class WorkspaceDatasourceFactory {
           );
 
         if (!dataSourceMetadata) {
-          throw new Error('Data source metadata not found');
+          throw new Error(
+            `Data source metadata not found for workspace ${workspaceId}`,
+          );
         }
 
         if (!cachedObjectMetadataCollection) {
-          throw new Error('Object metadata collection not found');
+          throw new Error(
+            `Object metadata collection not found for workspace ${workspaceId}`,
+          );
+        }
+
+        const cachedEntitySchemaOptions =
```

Comment:
we are caching the entity schema options (this is the typeOrmNaming, not sure it's super clear)